Libraries/JavaScriptAppEngine/polyfills/document.js

```diff
@@ -1,34 +1,32 @@
-/* eslint global-strict: 0 */
-(function(GLOBAL) {
-  /**
-   * The document must be shimmed before anything else that might define the
-   * `ExecutionEnvironment` module (which checks for `document.createElement`).
-   */
+/* eslint strict: 0 */
+/**
+ * The document must be shimmed before anything else that might define the
+ * `ExecutionEnvironment` module (which checks for `document.createElement`).
+ */
```

Comment:
@davidaurelio Should I add something like `// TODO: Remove document polyfill now that chrome debugging is in a web worker`?